src/main/java/org/gridsuite/cgmes/boundary/server/CgmesBoundaryController.java

```diff
@@ -47,6 +49,15 @@
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(boundaries);
     }
 
+    @GetMapping(value = "/boundaries/last", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get last boundary", response = List.class)
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The last EQ and TP boundaries")})
+    public ResponseEntity<List<BoundaryInfo>> getLastBoundaries() {
+        List<BoundaryInfo> boundaries = new ArrayList<>();
+        BOUNDARY_PROFILES.stream().forEach(profile -> boundaries.add(cgmesBoundaryService.getLastBoundary(profile)));
```

Comment:
Could be done with boundaries.stream().map()

Reply:
done